pages/login/index.js

```diff
@@ -30,64 +31,60 @@ export default function Login() {
 
   function updateLogin(e) {
     const { id, value } = e.target;
-    if (id != "checkbox") {
+    if (id !== "checkbox") {
       setLogin((prevValue) => {
         if (id === "email") {
           return { email: value, password: prevValue.password };
         }
+
         return { email: prevValue.email, password: value };
       });
     }
   }
 
   function authenticate(e) {
     e.preventDefault();
-    users.forEach((user) => {
-      if(login.email == user.email) {
-        console.log(true) 
-      }
-    })
+    let successfulLogin = users.find((user) => {
+      return login.email == user.email && login.password == user.password;
+    });
+
+    if (!successfulLogin) {
+      setSignInError(true);
+      return;
+    }
+
+    router.push("/home");
   }
 
   return (
     <div className="login-page">
       <div className="form-container">
-        <form className="container">
+        <form onSubmit={authenticate} type="submit" className="container">
```

Comment:
**issue:** `type="submit"` is meant for inputs that will trigger submission on the form.